recipes/cgal/all/conanfile.py

```diff
@@ -10,24 +12,11 @@ class CgalConan(ConanFile):
     homepage = "https://github.com/CGAL/cgal"
     description = "C++ library that aims to provide easy access to efficient and reliable algorithms"\
                   "in computational geometry."
-    topics = ("conan", "cgal", "geometry", "algorithms")
+    topics = ("cgal", "geometry", "algorithms")
     settings = "os", "compiler", "build_type", "arch"
     generators = "cmake"
     exports_sources = "CMakeLists.txt"
-    options = {
-        "with_cgal_core": [True, False],
-        "with_cgal_qt5": [True, False],
```

Comment:
Usually we don't remove options: https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#can-i-remove-an-option-from-a-recipe

But I see you replaced all versions, so I see no problem, as only old packages will keep those options. 👍